.github/ISSUE_TEMPLATE/---enhancement-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "\U0001F389 Enhancement request"
+about: Ask for something to be improved in this repo
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Please make sure that your request is in-scope for this repo.
+
+This project has a strictly limited scope: compiling Scala to JS, and interoperate with the ECMAScript language. Anything else is out of scope.
+
+Typical things that are out of scope include:
+
+* Can it be implemented as a library or as an external tool? If yes, it is out of scope.
+* Is it a missing Java library? If yes, check whether there is a separate library that provides it (e.g., `java.time`).
+* Is it a missing JavaScript library? Unless it is specified in ECMAScript itself, it is out of scope.
+* Is it a language feature request? Unless it addresses something of ECMAScript that cannot be done in Scala.js yet, it is out of scope.
+* Is it a request for more optimizations? Technically not out of scope, but we have already enough ideas and not enough time for those.
+
+Thank you for your understanding!
+-->
```

Comment:
I think for all of these it would be helpful to have alternatives (the bug section has this, and it comes off as much more helpful). Even if it is just: post on the mailing list or ask on the gitter channel if somebody has done this / knows something.